client/container_test.go

```diff
@@ -0,0 +1,73 @@
+package client
+
+import (
+	"context"
+	"testing"
+
+	"github.com/nspcc-dev/neofs-sdk-go/container"
+	"github.com/stretchr/testify/require"
+)
+
+func TestClient_Container(t *testing.T) {
+	c := newClient(t, nil, nil)
+
+	t.Run("missing signer", func(t *testing.T) {
+		tt := []struct {
+			name       string
+			methodCall func() error
+		}{
+			{
+				"put",
+				func() error {
+					_, err := c.ContainerPut(context.Background(), PrmContainerPut{cnrSet: true})
```

Comment:
Suggesting to derive `context.Background()` in separate var and pass `ctx`

Reply:
i'm cool, but you're cooler, fixed